README.md

```diff
@@ -165,15 +189,17 @@ $ ollama run llama2 "Summarize this file: $(cat README.md)"
  Ollama is a lightweight, extensible framework for building and running language models on the local machine. It provides a simple API for creating, running, and managing models, as well as a library of pre-built models that can be easily used in a variety of applications.
 ```
 
-### List models on your computer
+### Start Ollama
+
+If you installed Ollama using the installations on the website, you will usually not need to do this. If you built Ollama yourself, or you need to change some parameters, it may be necessary to run the service manually:
```

Comment:
This is definitely a bit confusing today. I think a more descriptive title might help. This is a great place to introduce `OLLAMA_HOST` as well (vs this paragraph)

Reply:
I was also thinking it would be good to have a doc called something like hosting... It can go over all the questions in the faq and show examples of ollama_host etc